### PR TITLE
[DTRA]/UPM-635/Ahmad/Arabic lang fix

### PR DESCRIPTION
**Pull request by ahmadtaimoor-deriv**
It includes the changes required to make the flutter chart work in rtl

### CODE REVIEW

**Review by review-deriv — COMMENTED** [review]
PR description is too short. You can use this format:

```
# Description
This is a summary of the changes. This should describe why do we need this change.
Whys and Hows come here.

# Cards
* https://clickup.com/12345
* https://clickup.com/12346

```

**Review by review-deriv — COMMENTED** [review]
PR description is too short. You can use this format:

```
# Description
This is a summary of the changes. This should describe why do we need this change.
Whys and Hows come here.

# Cards
* https://clickup.com/12345
* https://clickup.com/12346

```

**Review by review-deriv — COMMENTED** [review]
PR description is too short. You can use this format:

```
# Description
This is a summary of the changes. This should describe why do we need this change.
Whys and Hows come here.

# Cards
* https://clickup.com/12345
* https://clickup.com/12346

```

**Review by review-deriv — COMMENTED** [review]
PR description is too short. You can use this format:

```
# Description
This is a summary of the changes. This should describe why do we need this change.
Whys and Hows come here.

# Cards
* https://clickup.com/12345
* https://clickup.com/12346

```

**Review by review-deriv — COMMENTED** [review]
PR description is too short. You can use this format:

```
# Description
This is a summary of the changes. This should describe why do we need this change.
Whys and Hows come here.

# Cards
* https://clickup.com/12345
* https://clickup.com/12346

```

**Review by henry-deriv — APPROVED** [review]

**Review by suisin-deriv — APPROVED** [review]
LGTM

**Review by kate-deriv — APPROVED** [review]

File: sass/styles/main.scss [hunk 1]
```diff
@@ -412,6 +417,11 @@
     }
 }
 
+[dir='rtl'] .sc-input {
+    border-right: 1px solid $color-grey-5 !important;
```
Comment (maryia-deriv) [inline sass/styles/main.scss:421]: let's try getting rid of !important by finding a better place for this style.🙏🏻

**Review by maryia-deriv — CHANGES_REQUESTED** [review]

File: sass/styles/main.scss (hunk 1 above)
Comment (ahmadtaimoor-deriv) [reply sass/styles/main.scss:421]: Actually In the actual .sc-input class there is property **border-right: none !important;**  . I don't know if we can overwrite that without important .

**Review by maryia-deriv — APPROVED** [review]

**Pull request merged**